scout/server/blueprints/login/views.py

```diff
@@ -123,6 +123,48 @@ def users():
     return render_template("login/users.html", **data)
 
 
+@login_bp.route("/remove_user/<email>", methods=["GET"])
+def remove_user(email):
+    """Remove a user from the database."""
+    if not current_user.is_admin:
+        flash("You are not authorized to remove user accounts.", "warning")
+        return safe_redirect_back(request)
+
+    user_obj = store.user(email)
+    if not user_obj:
+        flash(f"User {email} not found in the database", "warning")
+        return safe_redirect_back(request)
+
+    if store.user_mme_submissions(user_obj):
+        flash(
+            f"User {email} has associated Matchmaker Exchange submissions "
```

Comment:
With a modal I guess one could replicate the "pick an alternate contact". But all good like this, its a convenience already.